### PR TITLE
Stylise psgdpr page

### LINKED ISSUE #24: [DESKTOP - My account Page] GDPR - Personal data
Epic: https://github.com/PrestaShop/theme-refacto/issues/2
Responsive's issue: https://github.com/PrestaShop/theme-refacto/issues/30

[Link to Figma prototype](https://www.figma.com/file/LfVl5leeSKcVUhSaYwhbtM/New-Theme?node-id=236%3A10538)

#### UX and design explanations
- A "download" icon has been added to both "Get my data to PDF" and "Get my data to CSV" button to emphasize that it's downloadable

#### Mockup
desktop - gdpr personal data

### LINKED ISSUE #30: [RESPONSIVE - My account Page] GDPR - Personal data
Epic: https://github.com/PrestaShop/theme-refacto/issues/2
Desktop's issue: https://github.com/PrestaShop/theme-refacto/issues/24

[Link to Figma prototype](https://www.figma.com/file/LfVl5leeSKcVUhSaYwhbtM/New-Theme?node-id=298%3A11931)

#### UX and design explanations
- A "download" icon has been added to both "Get my data to PDF" and "Get my data to CSV" button to emphasize that it's downloadable

#### Mockup
responsive - gdpr personal data

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Some style tweaks to be in accordance with mockups
| Type?             | refactor
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | closes #24 closes #30.
| How to test?      | Look at the gdpr page from your account

image
image

### CODE REVIEW

File: modules/psgdpr/views/templates/front/customerPersonalData.tpl [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <a id="exportDataToCsv" class="btn btn-outline-primary d-block d-sm-inline-block mx-0 mb-2 mb-sm-0" target="_blank" href="{$psgdpr_csv_controller|escape:'htmlall':'UTF-8'}"><i class="material-icons">download</i> {l s='GET MY DATA TO CSV' mod='psgdpr'}</a>
+    <a id="exportDataToPdf" class="btn btn-outline-primary d-block d-sm-inline-block mx-0 mx-sm-2" target="_blank" href="{$psgdpr_pdf_controller|escape:'htmlall':'UTF-8'}"><i class="material-icons">download</i> {l s='GET MY DATA TO PDF' mod='psgdpr'}</a>
+  </div>
+
+  <h2 class="fs-5">{l s='Rectification & Erasure requests' mod='psgdpr'}</h2>
+  <p>{l s='You have the right to modify all the personal information found in the "My Account" page. For any other request you might have regarding the rectification and/or erasure of your personal data, please contact us through our' mod='psgdpr'} <a href="{$psgdpr_contactUrl|escape:'htmlall':'UTF-8'}">{l s='contact page' mod='psgdpr'}</a>. {l s='We will review your request and reply as soon as possible.' mod='psgdpr'}</p>
+
+ {literal}
+ <script type="text/javascript">
```
Comment (Hlavtox) [inline modules/psgdpr/views/templates/front/customerPersonalData.tpl:37]: @NeOMakinG This should be changed in the core bro

Comment (NeOMakinG) [reply modules/psgdpr/views/templates/front/customerPersonalData.tpl:37]: But well I guess this module isn't that used by others modules/themes, we could!

File: modules/psgdpr/views/templates/front/customerPersonalData.tpl [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+     var psgdpr_front_controller = "{/literal}{$psgdpr_front_controller|escape:'htmlall':'UTF-8'}{literal}";
+     var psgdpr_id_customer = "{/literal}{$psgdpr_front_controller|escape:'htmlall':'UTF-8'}{literal}";
+     var psgdpr_ps_version = "{/literal}{$psgdpr_ps_version|escape:'htmlall':'UTF-8'}{literal}";
+ </script>
+ {/literal}
+ {/block}
+
```
Comment (matthieu-rolland) [inline modules/psgdpr/views/templates/front/customerPersonalData.tpl:44]: missing new line at end of file:

```suggestion
 
 
```

**Review by matthieu-rolland — APPROVED** [review]
lgtm

**Comment by NeOMakinG** [comment]
@Hlavtox I'm afraid if we change the core, it will be considered as a BC for past themes, should we merge this PR at least for hummingbird first?

**Comment by NeOMakinG** [comment]
@Hlavtox As the only blocker here is related to a question "Should we change it in the core?", let's merge this one and clear that template when it's done on core side, the theme can't wait for a core update on this anyway, we can be flexible

**Pull request merged**